### PR TITLE
feat (jans-cedarling): Load trusted issuers on startup on the background

### LINKED ISSUE #10004: feat (jans-cedarling): Lazy loading of Trusted IDP metadata
If there are hundreds of trusted IDPs, it may make sense to implement lazy loading of the JWKS endpoint until the Cedarling sees a token with the respective issuer.

-------------------

### Test cases and code coverage

- [ ] Write unit test to cover added/changed code
- [ ] Update integration tests to cover added/changed code

-------------------

### Document the changes

- [ ] task for updating user guides if needed
- [ ] task for updating installation and configuration guides if needed 
- [ ] task for updating developer documentation if needed
- [ ] task for updating technical documentation if needed

### PR DESCRIPTION
**Pull request by olehbozhok**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

### Description
Added loading trusted issuers on basctround after startup. If user specify bootstrap parameter.

#### Target issue
  
  
  
  
closes #10004

#### Implementation Details
  

- fixed some cargo problems
- added ValueObject `IssClaim` (store normalized value of iss). 
    - it allows easier to track iss claim value over the code and remove unnecessary normalization checks.
- added async background loading trusted issuer
    - CEDARLING_TRUSTED_ISSUER_LOADER_TYPE
    - CEDARLING_TRUSTED_ISSUER_LOADER_WORKERS
- removed unused code
 

-------------------
### Test and Document the changes
- [x] Static code analysis has been run locally and issues have been fixed
- [x] Relevant unit and integration tests have been added/updated
- [x] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

Please check the below before submitting your PR. The PR will not be merged if there are no commits that start with `docs:` to indicate documentation changes or if the below checklist is not selected.
- [ ] **I confirm that there is no impact on the docs due to the code changes in this PR.**

## Summary by CodeRabbit

* **New Features**
  * Configurable trusted-issuer loader with sync/async modes and worker-count control (default 1).
  * OpenID configuration can include a status_list_endpoint for status-list validation.
  * Several bootstrap properties now include explicit default behavior notes (e.g., signature/status validation, log/type, LOCK, dynamic config).

* **Documentation**
  * Bootstrap properties updated with explicit default-value annotations and minor formatting tweaks.

* **Improvements**
  * Enhanced concurrency and thread-safety for JWT validation, key management, and issuer indexing.
  * More consistent issuer representation and normalization across token flows.

### CODE REVIEW

**Comment by mo-auto** [comment]
Error: Hi @olehbozhok, You did not reference an open issue in your [PR](13125). I attempted to create an [issue](#13126) for you.    
 Please update that [issues'](#13126) title and body and make sure I correctly referenced it in the above PRs body.

**Comment by mo-auto** [comment]
### :white_check_mark: **Snyk checks have passed. No issues have been found so far.**

| Status | Scanner |  Critical |  High |  Medium |  Low | Total (0) |
|---|---|---|---|---|---|---|
| :white_check_mark: | Open Source Security| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/janssenproject/pr-checks/a43fea2c-4e29-471a-ae1b-599f786aadb3?refs=ae5f5f0985e2714f086866a50e7929e70237c851&source=prChecksComment) |

:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).

File: jans-cedarling/cedarling/src/common/policy_store.rs [hunk 1]
```diff
@@ -271,33 +273,46 @@ impl Default for &TrustedIssuer {
...
+    pub(crate) fn get_token_metadata(&self, token_name: &str) -> Option<&TokenEntityMetadata> {
         self.token_metadata.get(token_name)
     }
 
-    pub fn normalized_issuer(&self) -> String {
-        let issuer_url = self.oidc_endpoint.origin().ascii_serialization();
-        normalize_issuer(&issuer_url)
+    pub(crate) fn iss_claim(&self) -> IssClaim {
+        IssClaim::new(&self.oidc_endpoint.origin().ascii_serialization())
     }
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/common/policy_store.rs:316]: It is OK, it is designed to work like that

File: jans-cedarling/cedarling/src/jwt/mod.rs [hunk 2]
```diff
@@ -160,66 +161,32 @@ impl JwtService {
...
         // This ensures local JWKS-only configurations work correctly
-        if !has_trusted_issuers
+        // works only in synchronous mode loading
+        if loader.jwt_config.trusted_issuer_loader.is_sync()
+            && !key_service.has_keys()
             && jwt_config.jwt_sig_validation
             && let Some(jwks) = jwt_config.jwks.as_ref()
         {
             key_service.insert_keys_from_str(jwks)?;
         }
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/mod.rs]: It is intentional.

File: jans-cedarling/cedarling/src/bootstrap_config/decode.rs [hunk 3]
```diff
@@ -112,6 +114,9 @@ impl BootstrapConfig {
             })
             .transpose()?;
 
+        let trusted_issuer_workers = NonZeroUsize::new(raw.trusted_issuer_loader_workers)
+            .unwrap_or_else(|| unsafe { NonZeroUsize::new_unchecked(1) });
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/bootstrap_config/decode.rs]: It is OK in this case. In this case it is safe

File: jans-cedarling/cedarling/src/jwt/issuer_index.rs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        // TODO: Optimize this lookup to have O(1) complexity, to have index structure
+        let index = self.index.read().expect(MUTEX_POISONED_ERR);
+        for issuer_config in index.values() {
+            for (token_key, token_metadata) in &issuer_config.policy.token_metadata {
+                if token_metadata.entity_type_name == entity_type_name {
+                    return Some(token_key.to_string());
+                }
+            }
+        }
+        None
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/issuer_index.rs:74]: Thanks, but it won't be fixed in the current PR. So let's skip this.

File: jans-cedarling/cedarling/src/jwt/validation/validator_cache.rs [hunk 5]
```diff
@@ -3,20 +3,23 @@
...
 use jsonwebtoken::Algorithm;
+use std::borrow::Cow;
 use std::collections::HashMap;
 use std::fmt::Display;
 use std::hash::{DefaultHasher, Hash, Hasher};
 use std::sync::{Arc, RwLock};
 
 type CachedValidator = Arc<RwLock<JwtValidator>>;
+const MUTEX_POISONED_ERR: &str =
+    "JwtValidatorCache RwLock poisoned due to another thread panicking while holding the lock";
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/validation/validator_cache.rs:23]: In a normal use case, this error will never happen. 
Handling poisoned mutex is extremely hard. That's why the current implementation is OK.

File: jans-cedarling/cedarling/src/bootstrap_config/raw_config/config.rs [hunk 6]
```diff
@@ -305,6 +306,28 @@ pub struct BootstrapConfigRaw {
...
+    /// Number of concurrent workers to use when loading trusted issuers.
+    /// Applies to both SYNC (parallel loading during initialization) and ASYNC (parallel background loading) modes.
+    /// Default value is 1.
+    /// Zero will be treated as default value.
+    #[serde(
+        rename = "CEDARLING_TRUSTED_ISSUER_LOADER_WORKERS",
+        default,
+        deserialize_with = "deserialize_or_parse_string_as_json"
+    )]
+    pub trusted_issuer_loader_workers: usize,
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/bootstrap_config/raw_config/config.rs]: The current comment is OK, it depicts parsing logic.

**Comment by olehbozhok** [comment]
@coderabbitai looks like all comments resolved. Maybe you can approve the PR?

File: jans-cedarling/cedarling/src/jwt/mod.rs [hunk 7]
```diff
@@ -147,10 +148,10 @@ impl JwtService {
         trusted_issuers: Option<HashMap<String, TrustedIssuer>>,
         logger: Option<Logger>,
     ) -> Result<Self, JwtServiceInitError> {
-        let mut status_lists = StatusListCache::default();
-        let mut issuer_configs = HashMap::default();
-        let mut validators = JwtValidatorCache::default();
-        let mut key_service = KeyService::new();
+        let status_lists = Arc::new(StatusListCache::default());
```
Comment (dagregi) [inline jans-cedarling/cedarling/src/jwt/mod.rs]: question here why wrap `StatusListCache` in Arc when the field in this struct is already wrapped in Arc?

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 8]
```diff
@@ -0,0 +1,792 @@
+// This software is available under the Apache-2.0 license.
```
Comment (haileyesus2433) [inline jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:1]: I think using `.expect()` in concurrent code can cause panics. If the semaphore fails or mutex is poisoned, the entire service crashes. we should replace with proper error handling

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 9]
```diff
@@ -0,0 +1,792 @@
...
+    }
+
+    // Await all tasks to complete
+    for handle in handles {
+        handle.await_result().await;
+    }
+
+    loader.check_keys_loaded();
+
+    let errors = Arc::into_inner(errors)
```
Comment (haileyesus2433) [inline jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:160]: i think `Arc::into_inner()`  fails silently and  If other references exist, `Arc::into_inner()` returns `None` and will panic. i think we should use `Arc::try_unwrap()` with proper error handling.

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 10]
```diff
@@ -0,0 +1,792 @@
...
+                    LogEntry::new(BaseLogEntry::new_system_opt_request_id(
+                        LogLevel::WARN,
+                        None,
+                    ))
+                    .set_message(format!("Could not load trusted issuer: {issuer_id}"))
+                    .set_error(error.to_string()),
+                );
+                errors_clone
+                     .lock()
+                     .expect("failed to lock errors mutex while recording issuer loading failure - mutex may be poisoned due to panic in another thread")
```
Comment (haileyesus2433) [inline jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs]: If a thread panics while holding the mutex, subsequent operations will fail. i think we should find ways recover from the poisoning

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 11]
```diff
@@ -0,0 +1,792 @@
...
+            "HS256 key from OpenID configuration should be stored in key service"
+        );
+    }
+
+    /// Tests synchronous loading of a single trusted issuer with status validation enabled.
+    ///
+    /// Verifies that when `jwt_status_validation` is true, the status list cache is initialized
+    /// for the issuer and contains the expected status list.
+    #[tokio::test]
+    #[ignore = "Mock server issue with status list endpoint"]
```
Comment (dagregi) [inline jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs]: if we ignore this test then what use does it have?

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 12]
```diff
@@ -0,0 +1,792 @@
...
+    for handle in handles {
+        handle.await_result().await;
+    }
+
+    loader.check_keys_loaded();
+
+    let errors = Arc::into_inner(errors)
+         .expect("failed to extract errors Arc - other references may still exist indicating concurrent access during issuer loading")
+         .into_inner()
+         .expect("failed to extract errors from Mutex - mutex may be poisoned due to panic while holding lock");
```
Comment (haileyesus2433) [inline jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs]: same issue we should find ways to recover from poison

File: jans-cedarling/cedarling/src/bootstrap_config/decode.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (haileyesus2433) [reply jans-cedarling/cedarling/src/bootstrap_config/decode.rs]: i think we should refactor this i dont see the point of using unnecessary `unsafe` when it could be done safely and i think we can also use `.unwrap_or(NonZeroUsize::MIN) // MIN is always 1`

[the remark this answers is not in the document]
Comment (haileyesus2433) [reply jans-cedarling/cedarling/src/bootstrap_config/decode.rs]: i think we should also add input validation for worker count `NonZeroUsize` prevents zero, but no upper bound check exists. A misconfigured value (e.g., 10000) could spawn excessive tasks.

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 13]
```diff
@@ -0,0 +1,792 @@
...
+    if let Some(openid_config) = iss_config.openid_config.as_ref() {
+        key_service
+            .get_keys_using_oidc(openid_config, logger)
+            .await?;
+    }
+
+    Ok(())
+}
+
+#[cfg(test)]
```
Comment (haileyesus2433) [inline jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:281]: there are critical test cases missing like

1. **No test for true lazy loading**
   - Missing: Test that JWKS is fetched when token with unloaded issuer arrives
   - Missing: Test that validation waits for lazy load to complete

2. **No test for race conditions**
   - Missing: Test concurrent token validation during async loading
   - Missing: Test multiple tokens from same issuer during loading

3. **No test for WASM compatibility**
   - Missing: WASM-specific test cases
   - Missing: Test spawn_task behavior in WASM

4. **No test for error recovery**
   - Missing: Test behavior when JWKS fetch fails after retry
   - Missing: Test mutex poisoning recovery

5. **No test for worker limit**
   - Missing: Test behavior with maximum workers
   - Missing: Test behavior with single worker and many issuers

6. **No integration test for async mode timing**
   - Missing: End-to-end test showing async loading doesn't block startup
   - Missing: Test that service accepts requests while loading in background

File: jans-cedarling/cedarling/src/common/issuer_utils.rs [hunk 14]
```diff
@@ -3,10 +3,73 @@
...
+        Self(normalize_issuer(issuer))
+    }
+
+    /// Get the issuer as a &str
+    pub(crate) fn as_str(&self) -> &str {
+        &self.0
+    }
+}
+
+impl<'de> Deserialize<'de> for IssClaim {
```
Comment (dagregi) [inline jans-cedarling/cedarling/src/common/issuer_utils.rs:27]: is there a reason for doing this manually why not just derive the macro Deserialize and Serialize

File: jans-cedarling/cedarling/src/bootstrap_config/decode.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (dagregi) [reply jans-cedarling/cedarling/src/bootstrap_config/decode.rs]: > i think we should refactor this i dont see the point of using unnecessary `unsafe` when it could be done safely and i think we can also use `.unwrap_or(NonZeroUsize::MIN) // MIN is always 1`

I agree with @haileyesus2433 with this unless there is an explicit reason to use unsafe we should avoid it

File: jans-cedarling/cedarling/src/common/issuer_utils.rs (hunk 14 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/common/issuer_utils.rs:27]: Because using the derive macro won't be called `IssClaim::new`, the value won't be normalized.

File: jans-cedarling/cedarling/src/bootstrap_config/decode.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/bootstrap_config/decode.rs]: using `NonZeroUsize::MIN` is a really good idea. Thanks!

**Comment by olehbozhok** [comment]
> correct me if am wrong @olehbozhok but this PR implements async/sync loading modes but NOT true lazy loading as described in the issue. The issue states:
> 
> > "Implement lazy loading of the JWKS endpoint until the Cedarling sees a token with the respective issuer"
> 
> but current implementation still loads ALL issuers at startup (either synchronously or in the background). True lazy loading would:
> 
> * NOT fetch JWKS until a token from that issuer is encountered
> * Trigger on-demand fetching when validating a token from an unloaded issuer
> 
> there is a misalignment with the issue and the PR

I was thinking a lot about that. The main point of authorisation is that it should be incredibly fast. Any delay may cause a freeze of the UI application, which we would like to avoid. 
After communication with Michael, he agreed with me that if a user doesn't want to load all trusted issuers on start better to load them in the background.

In a usual use case, we have a small amount of trusted issuers. (it's words of Michael)

**Comment by haileyesus2433** [comment]
> > correct me if am wrong @olehbozhok but this PR implements async/sync loading modes but NOT true lazy loading as described in the issue. The issue states:
> > > "Implement lazy loading of the JWKS endpoint until the Cedarling sees a token with the respective issuer"
> > 
> > 
> > but current implementation still loads ALL issuers at startup (either synchronously or in the background). True lazy loading would:
> > 
> > * NOT fetch JWKS until a token from that issuer is encountered
> > * Trigger on-demand fetching when validating a token from an unloaded issuer
> > 
> > there is a misalignment with the issue and the PR
> 
> I was thinking a lot about that. The main point of authorisation is that it should be incredibly fast. Any delay may cause a freeze of the UI application, which we would like to avoid. After communication with Michael, he agreed with me that if a user doesn't want to load all trusted issuers on start better to load them in the background.
> 
> In a usual use case, we have a small amount of trusted issuers. (it's words of Michael)

ok well in that case i think the issue should be updated then so that it aligns with the PR dont u think so @olehbozhok?

File: jans-cedarling/cedarling/src/jwt/mod.rs (hunk 7 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/mod.rs]: thanks, fixed 7ede7b262b4dc86cab5b2e71ade71cb8b4d44701

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 8 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:1]: What do you mean by "proper error handling`?
Just return an error and propagate it? Or try to run the process of fetching data again?
Or maybe anything other?

From my point of view, if the mutex is poisoned or the semaphore is already closed, this is a critical error.
Because we made a logical error in how to work with workers. Probably, we can log this, but nothing more. We have no guarantee that the data that we have is valid or consistent.

Comment (haileyesus2433) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:1]: yes i think we if we add an Exponential Backoff strategy, then if it still fails we should log it instead of crashing the whole thing, i think they should be able to continue with loaded trusted issuers instead of crashing

File: jans-cedarling/cedarling/src/bootstrap_config/decode.rs [hunk 15]
```diff
@@ -115,7 +115,7 @@ impl BootstrapConfig {
             .transpose()?;
 
         let trusted_issuer_workers = NonZeroUsize::new(raw.trusted_issuer_loader_workers)
-            .unwrap_or_else(|| unsafe { NonZeroUsize::new_unchecked(1) });
+            .unwrap_or_else(|| NonZeroUsize::MIN);
```
Comment (dagregi) [inline jans-cedarling/cedarling/src/bootstrap_config/decode.rs]: remove the closure here clippy is throwing an error

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 8 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:1]: If it crashed once, it’s very likely to crash again on the next attempt due to an incorrect implementation. And it does not really make sense. So, throwing an error and a log message may be one possible way instead of panicking. But still, because it has a wrong implementation, we shouldn't continue execution... But we can try. 

So my decision is to log without retry. Maybe the log message will be sent to the LOCK server, and the developer will be notified.

**Comment by dagregi** [comment]
I just checked why all the CI is failing and apparently `cedar-policy-core` version 4.9.0 removed `Type::EntityOrRecord` so we either have to pin it to 4.8.2 or accomodate to the new changes but this is outside the scope of this pr

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 13 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:281]: Added unit tests that are reasonable.

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 12 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs]: resolved

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 8 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:1]: resolved

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 9 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:160]: resolved

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 10 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs]: resolved

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs (hunk 11 above)
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs]: resolved

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 16]
```diff
@@ -0,0 +1,842 @@
...
+    SemaphoreAcquire,
+    #[error(
+        "failed to extract errors Arc - other references may still exist indicating concurrent access during issuer loading"
+    )]
+    ErrorsArcExtraction,
+    #[error(
+        "failed to extract errors from Mutex - mutex may be poisoned due to panic while holding lock"
+    )]
+    ErrorsMutexExtraction,
+}
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:41]: in this case I will get warn:
type `jwt::trusted_issuers_loader::TrustedIssuerLoaderError` is more private than the item `jwt::error::JwtServiceInitError::LoadTrustedIssuers::0`

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 17]
```diff
@@ -0,0 +1,842 @@
...
+    }
+
+    if let Some(openid_config) = iss_config.openid_config.as_ref() {
+        key_service
+            .get_keys_using_oidc(openid_config, logger)
+            .await?;
+    }
+
+    Ok(())
+}
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:267]: resolved

File: jans-cedarling/bindings/cedarling_python/src/authorize/errors.rs [hunk 18]
```diff
@@ -129,8 +129,8 @@ macro_rules! errors_functions {
     ($($case_name:ident => $error_class:ident),*) => {
         // is used to map CedarlingAuthorizeError to python error
         #[allow(clippy::boxed_local)]
```
Comment (dagregi) [inline jans-cedarling/bindings/cedarling_python/src/authorize/errors.rs]: remove this allow since you remove the boxing below

File: jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs [hunk 19]
```diff
@@ -0,0 +1,842 @@
...
+
+    let errors = Arc::into_inner(errors)
+        .ok_or(TrustedIssuerLoaderError::ErrorsArcExtraction)?
+        .into_inner()
+        .map_err(|_| TrustedIssuerLoaderError::ErrorsMutexExtraction)?;
+
+    // if there were any errors, return the first one
+    if let Some(first_error) = errors.into_iter().next() {
+        return Err(first_error);
+    }
```
[the remark this answers is not in the document]
Comment (olehbozhok) [reply jans-cedarling/cedarling/src/jwt/trusted_issuers_loader.rs:170]: resolved

**Comment by olehbozhok** [comment]
@coderabbitai, if you don't have any suggestions, pls approve the PR

**Review by haileyesus2433 — APPROVED** [review]

**Review by moabu — APPROVED** [review]
